src/Aeon/Calendar/BusinessHours/BusinessHours.php

```diff
@@ -83,6 +83,17 @@ public function nextBusinessDay(Day $day, int $maximumDays = 365) : Day
         return $nextDay;
     }
 
+    public function nextBusinessDays(Day $day, int $businessDays, int $maximumDays = 365) : Day
```

Comment:
could you explain the logic behind this name?

Reply:
Hmm `give me nextBusinessDays($today, 5)` indicates that it should return the collection of business days. The name is wrong. It should be something more like `give me n-th business day from a given date`. I don't have satisfactory name for now (one idea: `nextBusinessDayIn`). I'll think about the better name for it.